aspnetcore/host-and-deploy/azure-apps/index.md

```diff
@@ -314,7 +318,7 @@ Use Visual Studio or the .NET Core CLI for a [self-contained deployment (SCD)](/
    dotnet publish --configuration Release --runtime win-x86 --self-contained
    ```
 
-1. Move the contents of the *bin/Release/{TARGET FRAMEWORK}/{RUNTIME IDENTIFIER}/publish* directory to the site in App Service. If dragging the *publish* folder contents from your local hard drive or network share directly to App Service in the Kudu console, drag the files to the `D:\home\site\wwwroot` folder in the Kudu console.
+1. Move the contents of the *bin/Release/{TARGET FRAMEWORK}/{RUNTIME IDENTIFIER}/publish* directory to the site in App Service. If dragging the *publish* folder contents from your local hard drive or network share directly to App Service in the [Kudu console](https://github.com/projectkudu/kudu/wiki), drag the files to the `D:\home\site\wwwroot` folder in the Kudu console.
```

Comment:
@bradygaster  I'm at the Kudu console. How do I drag n drop files?

Reply:
One can't drag-and-drop files in a terminal AFAIK. Maybe change the text to "copy?"

Reply:
Look at the link https://github.com/projectkudu/kudu/wiki/Kudu-console#upload-files-and-folder-using-drag-and-drop
you can drag and drop a zip file.  Should I delete the `D:\home\site\wwwroot` folder and then drag and drop my local `bin\Release\net5.0\win-x86\publish\wwwroot` folder zip file?

Reply:
Very minor item, missing "to":
Secure protocol bindings allow you specify a certificate 
Secure protocol bindings allow you to specify a certificate

Reply:
@wadepickett  please use  suggestion, it's easier to review and accept
![image](https://user-images.githubusercontent.com/3605364/98627516-8673e080-22b8-11eb-90a4-dd4018bce27e.png)